src/pages/creator/Onboard.tsx

```diff
@@ -0,0 +1,112 @@
+import { Web3Provider } from '@ethersproject/providers';
+import { useWeb3React } from '@web3-react/core';
+import { Link } from 'react-router-dom';
+import styled, { useTheme } from 'styled-components';
+
+import { PrimaryButton } from '../../components/Button';
+import { HeaderContentGapSpacer, HeaderSpacer } from '../../components/Header';
+import TwitterIcon from '../../components/icons/TwitterIcon';
+import { ContentWrapper, OutlinedContainer, PageContentWrapper, PageWrapper } from '../../components/layout/Common';
+import { TextField } from '../../components/TextField';
+import { Text } from '../../styles/typography';
+
+const OnboardTitle = styled.h1`
+  font-family: 'Scto Grotesk A';
+  font-weight: normal;
```

Comment:
this should be inherited and if we use a CSS reset it should be "normal" by default

Reply:
will fix typography stuff in a follow up pr